.github/workflows/go.yml

```diff
@@ -30,18 +34,66 @@ jobs:
           go-version: '${{ matrix.go }}'
 
       - name: 'Build'
+        if: ${{ matrix.arch == 'x64' }}
         run: go build -v ./...
 
       - name: 'Test'
+        if: ${{ matrix.arch == 'x64' }}
         run: go test -v ./...
 
+      - name: 'TestRace'
+        if: ${{ matrix.arch == 'x64' }}
+        run: go test -race -v ./...
+
+      - name: 'Bench'
+        if: ${{ matrix.arch == 'x64' }}
+        run: go test -run=- -bench=. -benchtime=1x -v ./...
+
+      - name: 'BenchRace'
+        if: ${{ matrix.arch == 'x64' }}
+        run: go test -run=- -bench=. -benchtime=1x -race -v ./...
+
       - name: 'Vet'
+        if: ${{ matrix.arch == 'x64' }}
         # -unsafeptr=false is needed because of the noescape function in bigint.go.
         run: go vet -unsafeptr=false ./...
 
       - name: 'Staticcheck'
         # staticcheck requires go1.14.
-        if: ${{ matrix.go != '1.13' }}
+        if: ${{ matrix.arch == 'x64' && matrix.go != '1.13' }}
         run: |
           go get honnef.co/go/tools/cmd/staticcheck
           staticcheck ./...
+
+      - name: 'GCAssert'
+        # Only run gcassert on latest version of Go. Inlining heuristics change
+        # from version to version.
+        if: ${{ matrix.arch == 'x64' && matrix.go == '1.17' }}
```

Comment:
Bleh, I feel like we won't remember to update this.

Reply:
Maybe change to >= for now? So at least it'll fail if the heuristics change and we get a version update.

Reply:
Done.